VikingQuiz.Api/Controllers/GoogleController.cs

```diff
@@ -10,20 +10,19 @@ namespace VikingQuiz.Api.Controllers
     [Route("api/[controller]")]
     public class GoogleController : Controller
     {
-        private readonly AuthenticationService authService;
+        private readonly AuthenticationService authenticationService;
         private readonly UserRepo repo;
```

Comment:
UserRepo need corect name like UserRepository, and parameter name not repo, more specific